tests/conftest.py

```
@@ -82,63 +81,73 @@ def ensure_test_schema() -> None:
+    user = result.scalar_one_or_none()
     if not user:
-        user = db.execute(select(User).where(User.username == "test_user")).scalar_one_or_none()
+        result = await db.execute(select(User).where(User.username == "test_user"))
+        user = result.scalar_one_or_none()
         if not user:
             user = User(username="test_user", created_at=datetime.now(UTC))
             db.add(user)
             try:
-                db.commit()
+                await db.commit()
             except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)